Example/Sources/ViewController.swift

```diff
@@ -106,7 +106,7 @@ extension ViewController: NDT7TestInteraction {
     }
 
     func measurement(origin: NDT7TestConstants.Origin, kind: NDT7TestConstants.Kind, measurement: NDT7Measurement) {
-        if let url = ndt7Test?.settings.url.hostname {
+        if let url = ndt7Test?.settings.hostname {
```

Comment:
This line is causing a build issue in the example project. Also, removal of the url object removes the possibility of getting data of the server's location